pkg/webhook/pod/mutating/sidecarset.go

```diff
@@ -79,9 +81,14 @@ func (h *PodCreateHandler) sidecarsetMutatingPod(ctx context.Context, req admiss
 		} else if !matched {
 			continue
 		}
+		// get user-specific revision or the latest revision of SidecarSet
+		suitableSidecarSet, err := h.getSuitableRevisionSidecarSet(&sidecarSet, pod)
```

Comment:
The ability to inject historical versions, specifically create scenarios. Whether special consider 'if create operation logic' is required considering internal code